### ISSUE TITLE
Help - Configuring terminal font

### ISSUE THREAD

**Issue by rogersavage** [CypherpunkArmory/UserLAnd#1190]
# Describe the issue you are having.

I want to configure the font in UserLAnd's terminal emulator.

# Describe how your task behaves in a regular Linux environment

I also run a Debian graphical desktop interface inside UserLAnd. I can configure different terminal emulators as usual, through their menus or through editing their config files.

However, what I want is to change the font in UserLAnd's terminal while running in console-only mode. I can't find an option in UserLAnd's interface, and I don't know if there's a file I can edit.

**Comment by Herzogs** [comment]
Hi, if you want change the font please continue this steps
1- `cd /data/data/com.termux/files/.termux`
2- `cp '/route/of/your-font.ttf' font.ttf`
3- restart userland

**Comment by macblain** [comment]
Ok, thank you very much.

Does that mean the font is hardcoded, otherwise? My goal is to have 80 columns in portrait mode. I have Xterm working exactly as I like it under LXDE, but I get better battery life and performance in console mode.

**Comment by papayura** [comment]
I still can't change extra-keys although i have added termux.properties in
/data/data/tech.ula/files/home/.termux . I need advice from you guys thanks.

**Comment by Herzogs** [comment]
Hello, You followed the steps that are described there https://github.com/CypherpunkArmory/UserLAnd/issues/617#issuecomment-477759175

**Comment by LawrenceKU03** [comment]
I tried it didn't work userland terminal display ubuntu 22

**Comment by LawrenceKU03** [comment]
![Screenshot_20231016-095309](https://github.com/CypherpunkArmory/UserLAnd/assets/80986362/2412012d-4ff4-4d6c-8e06-7b68815835d3)

**Comment by pedroall** [comment]
> Hi, if you want change the font please continue this steps
> 
> 1- `cd /host-rootfs/data/data/tech.ula/files/home/` 2- `mkdir -p .termux && cd .termux` 3- `cp /path/of/the/font.ttf font.ttf` 4- restart Userland

Does it still work? I followed all the steps and it didnt work

**Comment by rogersavage** [comment]
No, that was several versions ago unfortunately. No idea how to change it
now.

**Comment by andyrobreid** [comment]
For anyone ending up here, I managed by simply pinching. Mentioned in [this issue](https://github.com/CypherpunkArmory/UserLAnd/issues/1190#issuecomment-603285624)

**Comment by MNLierman** [comment]
> For anyone ending up here, I managed by simply pinching. Mentioned in [this issue](https://github.com/CypherpunkArmory/UserLAnd/issues/1190#issuecomment-603285624)

I appreciate the comment. That's not really a solution tho. I don't know what possessed the devs of these two, userland and termux, to use the world's most hard to read font. Seriously, this could win some awards. It's far from what one would describe as an easy to read font that minimizes eye strain and allows for rapid recognition of letters/words. It's a horrible font. The letter spacing is inconsistent, some letters get smashed together others don't, some letters have an uncommon boldness or thickness. The L's in particular look like blocks, and this alone causes strain on the unconscious mind when it's trying to rapidly decipher the text on the display. 

Ultimately, since it's uncommon in every day life to see letters designed like this, it makes it hard for the subconscious mind to quickly discern the meaning at first glance within milliseconds. One must consciously acknowledge how strange the letters are as they read back the lines, and the meaning is pieced together slowly because of how uncommon the appearance is. Perhaps if you are someone that is using this particular font all day long, then of course you would get used to it. Aside from that, I seriously just don't understand what would possess the developers to use such a font. There are so many better fonts out there, including free console fonts that are small in size and unrestricted.

**Comment by Herzogs** [comment]
I haven't used UserLand for a long time, but what I see is that it is now an integrated shell, so I think it is impossible to change the source.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
